ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/updates/TransactionRejections.scala

```diff
@@ -81,98 +60,43 @@ private[kvutils] object TransactionRejections {
         rejectionEntry.getSubmitterInfo,
       ),
       reasonTemplate = FinalReason(
-        errorVersionSwitch.choose(
-          V1.duplicateCommandsRejectionStatus(definiteAnswer, Code.ALREADY_EXISTS),
-          V2.duplicateCommandsRejectionStatus(definiteAnswer, existingCommandSubmissionId),
-        )
+        V2.duplicateCommandsRejectionStatus(definiteAnswer, existingCommandSubmissionId)
```

Comment:
V2 is redundant

Reply:
3b942886c4a14d5ae20d1ed8d663a58e6a1ee51d